skyportal/handlers/api/instrument.py

```diff
@@ -71,7 +75,6 @@ def post(self):
         if field_data is not None:
             if field_region is None:
                 return self.error('`field_region` is required with field_data')
```

Comment:
I think in this case backticks aren't the best choice. I would go with double quotes.

Reply:
I am removing these quotes for now, they arent useful.